### PR TITLE
Adds a pulp-ci-centos and pulp image based on it.

### PR DESCRIPTION
**Pull request by dkliban**
fixes: #8035
https://pulp.plan.io/issues/8035

### CODE REVIEW

File: .github/workflows/pr_build_images.yaml [hunk 1]
```diff
@@ -14,12 +14,15 @@ jobs:
             test_image: pulp-fedora31
           - test_type: pulp_galaxy_ng
             test_image: pulp-galaxy-ng
+          - test_type: pulp
+            test_image: pulp
```
Comment (fao89) [inline .github/workflows/pr_build_images.yaml:18]:
```suggestion
          - test_type: pulp_ci_centos
            test_image: pulp-ci-centos
```

File: .github/workflows/pr_build_images.yaml [hunk 2]
```diff
@@ -14,12 +14,15 @@ jobs:
...
     steps:
       - uses: actions/checkout@v2
       - name: Build the images
         run: |
           wget https://github.com/just-containers/s6-overlay/releases/download/v1.22.1.0/s6-overlay-amd64.tar.gz
           docker build --file pulp_ci/Containerfile --tag pulp/pulp-ci:latest .
+          docker build --file pulp_ci_centos/Containerfile --tag pulp/pulp-ci-centos:latest .
```
Comment (fao89) [inline .github/workflows/pr_build_images.yaml:25]: you don't need this line

**Review by fao89 — APPROVED** [review]

File: .github/workflows/pr_build_images.yaml (hunk 1 above)
Comment (dkliban) [reply .github/workflows/pr_build_images.yaml:18]: I would like us to publish an image called pulp/pulp-ci-centos and pulp/pulp. I also want to continue publishing the pulp/pulp-ci and pulp/pulp-fedora31 images.

Comment (fao89) [reply .github/workflows/pr_build_images.yaml:18]: oh! now I got it! yeah, you are right!

**Pull request merged**